app/views/candidate_interface/application_form/submit_success.html.erb

```diff
@@ -0,0 +1,12 @@
+<% content_for :title, page_title('Submit application') %>
+
+<div class="govuk-panel govuk-panel--confirmation govuk-!-margin-bottom-8">
+  <h1 class="govuk-panel__title">
+    Application successfully submitted
+  </h1>
+
+  <div class="govuk-panel__body">
+    Your application number is ?????
+<!--    TODO: provide correct application number. -->
```

Comment:
Maybe just remove this `<div>` for now while we iterate and decide what to show here